packages/wallet-ui/src/store/user/user.ts

```diff
@@ -99,6 +97,7 @@ export const loginUser = (
   ethAddress: string,
   signer: JsonRpcSigner,
   history: History,
+  // @ts-ignore
```

Comment:
You're being forced to use `@ts-ignore` because of the `@types/web3` package not being in `devDependencies`. This will go away when you fix that :)